heron/common/src/java/com/twitter/heron/common/basics/FileUtils.java

```diff
@@ -123,4 +123,17 @@ public static boolean isOriginalPackagePex(String packageFilename) {
   public static String getBaseName(String file) {
     return new File(file).getName();
   }
+
+  public static String getPkgType(String topologyBinaryFile) {
```

Comment:
actually, returning enum might not be what we want, since we need to put value as string into config. See: https://github.com/twitter/heron/pull/1599/files#diff-1043de1ec8f8e4c0194d4e6035e654cfL80